server/validation/validation_test.go

```diff
@@ -0,0 +1,425 @@
+package validation
+
+import (
+	"testing"
+
+	"github.com/openfga/openfga/pkg/tuple"
+	"github.com/openfga/openfga/pkg/typesystem"
+	"github.com/stretchr/testify/require"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+)
+
+func TestValidateTuple(t *testing.T) {
+
+	tests := []struct {
+		name        string
+		tuple       *openfgapb.TupleKey
+		model       *openfgapb.AuthorizationModel
+		expectError bool
+	}{
+		{
+			name:        "Malformed Object 1",
```

Comment:
rename tests here to use underscore pattern

e.g. `malformed_object_1` which will make it easier to copy/paste and search the test.